### PR TITLE
UI Overhaul

### PR DESCRIPTION
**Pull request by Lewis-Kelley**
screenshot-20180501 103831

* Changed the UI to be light-themed rather than...blue-themed?
* Made the buttons look a lot nicer
* Improved the scaling for everything (they now scale live rather than on page load)
* The aspect ratio of the play area is now fixed rather than scaling with the window shape
* Pulled the timer into a separate file (`timer.js`)
* Pulled out gesture code into a separate file and put it under test (`gesture.js`)
* I didn't modify the popover since that's being removed in the PR #67
* Moved a lot of code from `index.css` to `game.css` because...that's where they should be...

### CODE REVIEW

File: 2D Simulation/game.html [hunk 1]
```diff
@@ -20,180 +21,139 @@
...
+        Instructions:
+        <input id="txt_instruction" type="text" />
+      </label>
+      
+      <button class="button"
+              id="buttonEnter"
+              onclick="send_user_message_to_server(currentConfig); inputlength()">
+        Enter
+      </button>
+      <div class="rightInstruction" style="float: right;">
```
Comment (kgroble) [inline 2D Simulation/game.html:56]: I believe the `rightInstruction` class does not exist, so the class specifier does nothing. We could remove the class specifier or create the class and move the style elements there. There are a few "classes" where this happens.

**Review by kgroble — APPROVED** [review]
I believe I found an issue, but it was already existing and not critical. Overall this is definitely an improvement to the looks.

File: 2D Simulation/timer.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            clearInterval(x);
+            endGame();
+        } else {
+            $("#timer").html(minutes + ":" + padTens(seconds));
+        }
+
+    }, 1000);
+}
+
+function padTens(num, length) {
```
Comment (jcolemang) [inline 2D Simulation/timer.js]: What is `length` for here?

**Comment by jcolemang** [comment]
I would have the slightest preference towards making the background plain white but other than that it looks good

**Review by jcolemang — APPROVED** [review]

**Pull request merged**